### ISSUE TITLE
Infinite template recursion for 11-element tuples

### ISSUE THREAD

**Issue by eschnett** [USCiLab/cereal#175]
I believe this patch should be applied:

```diff
--- include/cereal/types/tuple.hpp
+++ include/cereal/types/tuple.hpp
@@ -63,7 +63,7 @@
     template <size_t Q, size_t R, char ... C>
     struct to_string_impl
     {
-      using type = typename to_string_impl<Q/1, Q%10, R+'0', C...>::type;
+      using type = typename to_string_impl<Q/10, Q%10, R+'0', C...>::type;
     };
 
     //! Base case with no quotient
```

**Comment by erichkeane** [comment]
You are absolutely correct!  This seems to work perfectly in the testing I've been doing in all cases I could think of (tuple <10, tuple >10, tuple >100), whereas the 11th tuple element will cause errors on the existing implementation.  eschnett: wanna submit this as a pull-request, that way Shane could pull this in?

**Comment by eschnett** [comment]
Done; this is now #176.

**Comment by AzothAmmo** [comment]
Merged to develop with some unit tests.  Anything else that gets fixed between now and the end of March will be released as 1.1.1.
